core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.spec.ts

```diff
@@ -28,12 +28,14 @@ import { SiteAnalyticsService } from 'services/site-analytics.service';
 import { ThreadDataBackendApiService } from 'pages/exploration-editor-page/feedback-tab/services/thread-data-backend-api.service';
 import { UserService } from 'services/user.service';
 import { UserInfo } from 'domain/user/user-info.model';
+import { RteOutputDisplayComponent } from 'rich_text_components/rte-output-display.component';
 
 class MockChangeDetectorRef {
   detectChanges(): void {}
 }
 
-describe('Translation Suggestion Review Modal Component', function() {
+// eslint-disable-next-line oppia/no-test-blockers
```

Comment:
Why is this needed?

Reply:
This is here for testing purposes only, will remove once ready for review